src/blog/2023/007-form-js-1.5.md

```diff
@@ -1,23 +1,22 @@
 ---
 
-title: Empowering complex forms with form-js@1.x
-description: "Recapping our recent form-js releases: form layouting, flexible data binding, and FEEL expression properties to simplify complex form development"
+title: Building Complex Forms
+description: "Recapping our recent form-js releases: Form layouting, flexible data binding, and FEEL expression properties to simplify complex form development"
 preview_image: http://bpmn.io/assets/attachments/blog/2023/007-group-path.png
 layout: blogpost.hbs
-slug: 2023-form-js-1.5-recent-developments
+slug: 2023-form-js-1-5-0-building-complex-forms
```

Comment:
Does the blog system we use handle redirects when we rename the slug? The old one should redirect with 301

Reply:
Did we already publish the old one, i.e. market externally?

Reply:
We don't handle redirects, but could make them work with a little effort.

Reply:
Added redirect handling via https://github.com/bpmn-io/bpmn.io/pull/177/commits/62bf9acb49c9de43c802af5b3a25c8382e299e83.

Now re-directing from old to new slug.